mappings/net/minecraft/block/Vines.mapping

```diff
@@ -0,0 +1,4 @@
+CLASS net/minecraft/class_4949 net/minecraft/block/Vines
```

Comment:
`VineHelper` maybe? just `Vines` seems too ambiguous to me

Reply:
i use this name as we have Texts class. this holds a collision shape and an air check for root for those who want to suggest alternative names.